server/test/unit/server/webservices/views/test_repositories.py

```diff
@@ -0,0 +1,1977 @@
+        )
+        sync_schedule = RepoSyncSchedulesView()
+        try:
+            sync_schedule.post(mock_request, 'mock_repo', 'mock_importer')
+        except pulp_exceptions.UnsupportedValue, response:
+            pass
+        else:
+            raise AttributeError("UnsupportedValue should be raised with extra fields in body")
+
+        self.assertEqual(response.http_status_code, 400)
+        self.assertTrue(response.error_code is error_codes.PLP0017)
+
+
+class TestRepoSyncScheduleResourceView(unittest.TestCase):
+    """
+    Tests for the RepoSyncScheduleResourceView.
+    """
+
+    @mock.patch('pulp.server.webservices.controllers.decorators._verify_auth',
+                new=assert_auth_READ())
+    @mock.patch('pulp.server.webservices.views.repositories.RepoSyncScheduleResourceView._get')
+    @mock.patch(
+        'pulp.server.webservices.views.repositories.manager_factory.repo_sync_schedule_manager')
+    @mock.patch('pulp.server.webservices.views.repositories.reverse')
+    def test_get_sync_schedule(self, mock_rev, mock_manager, mock_get):
+        """
+        Retrieve a single schedule.
+        """
+
+        mock_request = mock.MagicMock()
+        sync_resource = RepoSyncScheduleResourceView()
+        response = sync_resource.get(mock_request, 'mock_repo', 'mock_importer', 'mock_schedule')
+        sync_resource.manager.validate_importer.assert_called_once_with('mock_repo',
+                                                                        'mock_importer')
+        mock_get.assert_called_once_with('mock_schedule', mock_rev.return_value)
+        self.assertTrue(response is mock_get.return_value)
+
+    @mock.patch('pulp.server.webservices.controllers.decorators._verify_auth',
+                new=assert_auth_DELETE())
+    @mock.patch('pulp.server.webservices.views.repositories.generate_json_response')
+    @mock.patch(
+        'pulp.server.webservices.views.repositories.manager_factory.repo_sync_schedule_manager')
+    def test_delete_sync_schedule(self, mock_manager, mock_resp):
+        """
+        Delete a single schedule.
+        """
+
+        mock_request = mock.MagicMock()
+        sync_resource = RepoSyncScheduleResourceView()
+        response = sync_resource.delete(mock_request, 'mock_repo', 'mock_importer', 'mock_schedule')
+        sync_resource.manager.delete.assert_called_once_with(
+            'mock_repo', 'mock_importer', 'mock_schedule')
+        mock_resp.assert_called_once_with(None)
+        self.assertTrue(response is mock_resp.return_value)
+
+    @mock.patch('pulp.server.webservices.controllers.decorators._verify_auth',
+                new=assert_auth_DELETE())
+    @mock.patch('pulp.server.webservices.views.repositories.generate_json_response')
+    @mock.patch(
+        'pulp.server.webservices.views.repositories.manager_factory.repo_sync_schedule_manager')
+    def test_delete_sync_schedule_invalid_schedule(self, mock_manager, mock_resp):
```

Comment:
I am not sure what you mean, I think that this is that test?

It is also covered by the hand testing:
invalid schedule_id
*includes bugfix [redmine-756](https://pulp.plan.io/issues/756)
http://fpaste.org/197622/14262601/